### PR TITLE
chore(demo): fix scroll position on page navigation

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [X] Documentation content changes

## What is the current behavior?
1. Open any long documentation page (e.g., https://tinkoff.github.io/maskito/core-concepts/processors)
2. Scroll to the end of the page
3. Press link to another documentation page

https://user-images.githubusercontent.com/35179038/214326914-0a4b7996-0d57-4b5a-95aa-2b71ec948eaf.mov

## What is the new behavior?

https://user-images.githubusercontent.com/35179038/214326747-09331bf8-d54b-4226-b1cf-42858368e316.mov

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/app/app.routes.ts [hunk 1]
```diff
@@ -136,8 +136,9 @@ export const appRoutes: Routes = [
 @NgModule({
     imports: [
         RouterModule.forRoot(appRoutes, {
-            initialNavigation: 'enabled',
```
Comment (nsbarsukov) [inline projects/demo/src/app/app.routes.ts:139]: deprecated since Angular 11

File: projects/demo/src/app/app.routes.ts [hunk 2]
```diff
@@ -136,8 +136,9 @@ export const appRoutes: Routes = [
...
+            initialNavigation: 'enabledBlocking',
             relativeLinkResolution: 'corrected',
+            scrollPositionRestoration: `enabled`,
```
Comment (nsbarsukov) [inline projects/demo/src/app/app.routes.ts:141]: This option will be the default in the future Angular.

Comment (splincode) [reply projects/demo/src/app/app.routes.ts:141]: please, also add to `taiga-ui.dev` that option

Comment (nsbarsukov) [reply projects/demo/src/app/app.routes.ts:141]: @splincode okey, I will add it to `taiga` and `angular-open-source-starter`

**Pull request merged**